scripts/release.sh

```diff
@@ -32,23 +33,23 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
   VERSION=$VERSION npm run build
 
   # update packages
-  cd packages/vue-template-compiler
-  npm version $VERSION
+  ( cd packages/vue-template-compiler
+  npm version "$VERSION"
   if [[ -z $RELEASE_TAG ]]; then
     npm publish
   else
-    npm publish --tag $RELEASE_TAG
+    npm publish --tag "$RELEASE_TAG"
   fi
   cd -
```

Comment:
In any case, for consistency, shouldn't this `cd -` be replaced as well?

Reply:
Indeed. Will do. I'm not sure if shellcheck would have spotted it after running it a second time. Will try. If not, we may found a bug....

Reply:
Opened [an issue](https://github.com/koalaman/shellcheck/issues/1146). :)